Atcoder_#359/b_couples.rb

```diff
@@ -0,0 +1,18 @@
+n = gets.to_i
+a = gets.split.map(&:to_i)
+
+positions = Hash.new { |hash, key| hash[key] = [] }
+
+a.each_with_index do |color, index|
```

Comment:
```suggestion
a.each.with_index do |color, index|
```
多分、最新の記法は、 each.with_index を使いましょうと rubocop で言われる気がすす。

Reply:
ありがとうございます！
こちら修正致しました！
53158220cac95e50f98d3350925cd6f44c8a5117

また調べた所、
`each.with_index`
の方が開始行が決められるなど汎用性が高いことがわかりました
https://qiita.com/gogotakataka1234/items/aeec8e301426282284ec